### ISSUE TITLE
Application crashes after setting SystemBackdrop on an empty window (v1.3-preview1)

### ISSUE THREAD

**Issue by plonkaadam** [microsoft/microsoft-ui-xaml#8364]
### Describe the bug

The application crashes immediately without visible exception after attempting to set `SystemBackdrop` on a window with no content.

### Steps to reproduce the bug

1. Create new application
2. Set SystemBackdrop in MainWindow constructor:

```
this.InitializeComponent();
SystemBackdrop = new MicaBackdrop();
```

3. Remove all the content from MainWindow.xaml `<Window />` tag:

```
<Window
    x:Class="App.MainWindow"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    mc:Ignorable="d">
</Window>
```

4. Try to run the application, it stops immediately with error code 0xc000027b.

### Expected behavior

Application attempts to subscribe to `Content.ActualThemeChanged` event but the content is not a FrameworkElement.
It should either wait for the content to be set or throw a managed exception with a friendly description.

### NuGet package version

Windows App SDK 1.3 Preview 1: 1.3.230228005-preview1

### Packaging type

Packaged (MSIX), Unpackaged

### Windows version

Windows 11 version 22H2 (22621, 2022 Update)

### IDE

Visual Studio 2022-preview

### Additional context

This is the info from the process dump file:

```
CONTEXT:  (.ecxr)
rax=000000188bf9bb40 rbx=000000188bf9c120 rcx=000000188bf9bb40
rdx=000000188bf9bff0 rsi=000001a903b9ad01 rdi=000000188bf9bb40
rip=00007ffde06d5a12 rsp=000000188bf9ba60 rbp=0000000000000001
 r8=0000000000000000  r9=000000188bf9bfa0 r10=0000000000000000
r11=000000188bf9c120 r12=00000000000004b1 r13=0000000080070057
r14=0000000000000002 r15=0000000000000001
iopl=0         nv up ei pl nz na pe nc
cs=0033  ss=002b  ds=002b  es=002b  fs=0053  gs=002b             efl=00000200
KERNELBASE!RaiseFailFastException+0x152:
00007ffd`e06d5a12 0f1f440000      nop     dword ptr [rax+rax]
Resetting default scope

EXCEPTION_RECORD:  (.exr -1)
ExceptionAddress: 00007ffcd35748bb (Microsoft_ui_xaml!DirectUI::SystemBackdrop::PerTargetConfigurationEntry::AttachActualThemeChanged+0x00000000000000b3)
   ExceptionCode: c000027b
  ExceptionFlags: 00000001
NumberParameters: 2
   Parameter[0]: 000001a903b9ad00
   Parameter[1]: 0000000000000001

PROCESS_NAME:  App3.exe

ERROR_CODE: (NTSTATUS) 0xc000027b - An application-internal exception has occurred.

EXCEPTION_CODE_STR:  8000ffff

EXCEPTION_PARAMETER1:  000001a903b9ad00

EXCEPTION_PARAMETER2:  0000000000000001

FAULTING_THREAD:  ffffffff

STACK_TEXT:  
00000018`8bf9ba60 00007ffd`e06d5a12 KERNELBASE!RaiseFailFastException+0x152
00000018`8bf9c040 00007ffd`e21dd0c9 combase!RoFailFastWithErrorContextInternal2+0x4d9
00000018`8bf9c2b0 00007ffc`d35748bb Microsoft_ui_xaml!DirectUI::SystemBackdrop::PerTargetConfigurationEntry::AttachActualThemeChanged+0xb3
00000018`8bf9c330 00007ffc`d357459e Microsoft_ui_xaml!DirectUI::SystemBackdrop::PerTargetConfigurationEntry::PerTargetConfigurationEntry+0x31e
00000018`8bf9c470 00007ffc`d3574efa Microsoft_ui_xaml!DirectUI::SystemBackdrop::InvokeOnTargetConnected+0xaa
00000018`8bf9c4e0 00007ffc`d35e76ca Microsoft_ui_xaml!DirectUI::DesktopWindowXamlSource::put_SystemBackdropImpl+0x156
00000018`8bf9c550 00007ffc`d36fcb61 Microsoft_ui_xaml!DirectUI::DesktopWindowImpl::put_SystemBackdropImpl+0x21
00000018`8bf9c580 00007ffc`d36cddb1 Microsoft_ui_xaml!DirectUI::Window::put_SystemBackdropImpl+0x41
00000018`8bf9c5b0 00007ffc`d34f84ff Microsoft_ui_xaml!ctl::interface_forwarder<ABI::Microsoft::UI::Xaml::IWindow2,DirectUI::WindowGenerated>::put_SystemBackdrop+0x3f
00000018`8bf9c5e0 00007ff5`d0128c9d unknown!unknown+0x0
00000018`8bf9c5e8 000001a9`67e5bff0 unknown!unknown+0x0
00000018`8bf9c5f0 000001a9`03b733b8 unknown!unknown+0x0
00000018`8bf9c5f8 00000018`8bf9ca60 unknown!unknown+0x0
00000018`8bf9c600 00007ff5`d0b62600 unknown!unknown+0x0
00000018`8bf9c608 00000018`8bf9c690 unknown!noop+0x0
00000018`8bf9c610 00004e7e`6ae5a1ce unknown!unknown+0x0
00000018`8bf9c618 00007ff6`2fcef108 App3_exe!vtable_InlinedCallFrame+0x0
00000018`8bf9c620 00000018`8bf9d0c8 unknown!unknown+0x0
00000018`8bf9c628 0000fff9`a69f0981 unknown!unknown+0x0
00000018`8bf9c630 0000fff9`a69f0981 unknown!unknown+0x0
00000018`8bf9c638 00000018`8bf9c5e0 unknown!unknown+0x0
00000018`8bf9c640 00007ff5`d0128c9d unknown!unknown+0x0
00000018`8bf9c648 00000018`8bf9c690 unknown!noop+0x0
00000018`8bf9c650 00000018`8bf9c728 unknown!unknown+0x0
00000018`8bf9c658 0000fff9`a69f0981 unknown!unknown+0x0
00000018`8bf9c660 00000018`8bf9c910 unknown!unknown+0x0
00000018`8bf9c668 00000018`8bf9ca60 unknown!unknown+0x0
00000018`8bf9c670 00007ff5`d0b62600 unknown!unknown+0x0
00000018`8bf9c678 000001a9`67ee3808 unknown!unknown+0x0
00000018`8bf9c680 00000000`00000001 unknown!unknown+0x0
00000018`8bf9c688 00000000`00000130 unknown!unknown+0x0


SYMBOL_NAME:  Microsoft_ui_xaml!DirectUI::SystemBackdrop::PerTargetConfigurationEntry::AttachActualThemeChanged+b3

MODULE_NAME: Microsoft_ui_xaml

IMAGE_NAME:  Microsoft.ui.xaml.dll

STACK_COMMAND:  *** Stowed Exception v2 ***; .exr -1; dpp 0x1a903b9ad00 L0x1; dt 0x1a9038fe508 combase!STOWED_EXCEPTION_INFORMATION_V2 -r; dps 0x1a9038fd4f0 L0x8 ; ** Pseudo Context ** StowedPseudo ** Value: 27 ** ; kb

FAILURE_BUCKET_ID:  STOWED_EXCEPTION_8000ffff_Microsoft.ui.xaml.dll!DirectUI::SystemBackdrop::PerTargetConfigurationEntry::AttachActualThemeChanged

OS_VERSION:  10.0.22621.1

BUILDLAB_STR:  ni_release

OSPLATFORM_TYPE:  x64

OSNAME:  Windows 10

IMAGE_VERSION:  3.1.3.0

FAILURE_ID_HASH:  {4e19387d-8e1a-2380-4807-46ef3df1108e}
```

**Comment by bpulliam** [comment]
This issue has been fixed in 1.3.230502000.

**Comment by bpulliam** [comment]
1.3.1 = 1.3.230502000 (built May 2, 2023)
